clarifai/client/client.py

```diff
@@ -739,3 +740,82 @@ def _base64_encoded_data_op(self, data, op):
     headers = self._get_json_headers()
     response = self._get_json_response(url, data=data, headers=headers)
     return self._parse_response(response)
+
+  def tag_color(self, files, model=None, local_ids=None, meta=None,
```

Comment:
Maybe just color(). 

"model", "select_classes", and "language" are not supported so please remove from args and the docs below. You can juts pass in their defaults of None to the _multi_data_op call for now.